msg/EstimatedRgvState.msg

```diff
@@ -1,14 +1,15 @@
-## Constants
-# Possible values of the `rgv_id` field
-int8 RGV_1 = 1
-int8 RGV_2 = 2
-
 ## Fields
-# Time that this estimate is for
+# Time that these estimates are for
 time timestamp
-# ID of the measured RGV
-int8 rgv_id
-# Estimated position of the RGV
-float64[3] position
-# Estimated velocity of the RGV
-float64[3] velocity
\ No newline at end of file
+# Estimated position of RGV 1 in the local inertial frame
+float64[3] rgv1_position
+# Estimated velocity of RGV 1 in the local inertial frame
+float64[3] rgv1_velocity
```

Comment:
Is a full velocity vector more than we need? If this were just speed or still/moving that would be enough for the state machine criteria generator. Does Lyon need velocity for waypoint generation? Is velocity more difficult to predict than just speed or still/moving?

Reply:
can we call `rgv_1position`, etc. something according to the naming convention or at least include the frame/ that it's a local position